setup.cfg

```diff
@@ -0,0 +1,13 @@
+[isort]
+line_length=80
+skip=.git,.tox,docs
+
+[pycodestyle]
+count=True
+exclude=.git,.tox,docs,__pycache__
+max-line-length=80
+statistics=True
+
+[pydocstyle]
+ignore=D105,D400,D401
```

Comment:
For reference: 
http://www.pydocstyle.org/en/latest/error_codes.html

    Missing docstrings/Docstring Content Issues
    D105 	Missing docstring in magic method
    D400 	First line should end with a period
    D401 	First line should be in imperative mood